openslides_backend/action/actions/meeting_user/mixin.py

```diff
@@ -10,6 +11,77 @@
 
 
 class MeetingUserMixin(Action):
+    standard_fields = [
+        "comment",
+        "number",
+        "structure_level",
+        "vote_weight",
+        "personal_note_ids",
+        "speaker_ids",
+        "supported_motion_ids",
+        "submitted_motion_ids",
+        "assignment_candidate_ids",
+        "vote_delegated_vote_ids",
+        "vote_delegated_to_id",
+        "vote_delegations_from_ids",
+        "chat_message_ids",
+    ]
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        """ standard_fields have to be checked, but also it is not permitted, create an empty meeting_value instance"""
+        if any(fname in self.standard_fields for fname in instance.keys()) or not any(fname in ["about_me", "group_ids"] for fname in instance):
+            return super().check_permissions(instance)
+
+        def get_user_and_meeting_id() -> Tuple[int]:
+            fields = ["user_id", "meeting_id"]
+            if any(fname not in instance for fname in fields):
+                mu = self.datastore.get(
+                    fqid_from_collection_and_id("meeting_user", instance["id"]),
+                    ["user_id", "meeting_id"],
+                    lock_result=False,
+                )
+                return (mu[fname] for fname in fields)
+            return (instance[fname] for fname in fields)
+
+        def get_request_user_data() -> Dict[str, Any]:
+            return self.datastore.get(
+                fqid_from_collection_and_id("user", self.user_id),
+                ["organization_management_level", "committee_management_ids"],
+                lock_result=False,
+            )
+
+        def get_committee_id() -> int:
```

Comment:
same here

Reply:
Conversion with Joshua: The outer method is a lot easier to understand with such "named calls"